docs/5.x/writing-a-record-builder.md

```diff
@@ -0,0 +1,250 @@
+---
+category: Develop
+---
+
+<div markdown="1" class="alert alert-warning">
+**This API is unstable.**
+
+The RecordBuilder API will eventually be public and the only way to define archiving logic, but for now the API is unstable
+and subject to change. Please be aware it could potentially change between minor version releases.
+</div>
```

Comment:
Is this still unstable or is this now the default and we can remove this warning?

Reply:
that is up to the core team. feel free to change it yourself when your team decides the answer to that.